kickoff/templates/includes/devedition_release.html

```diff
@@ -0,0 +1,82 @@
+<script type='text/javascript'>
+URLToPartial="/cron/partial.json";
```

Comment:
I think we can leave it as is until we get better understanding about Socorro changes. There were some discussions on how to distinguish beta and devedition, but I'm not sure if it's firm yet. It kind of "works": doesn't suggest anything. What do you think?